crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -2008,6 +2008,7 @@ impl<'db> TypeInferenceBuilder<'db> {
 
     fn infer_augmented_op(
         &mut self,
+        db: &'db dyn Db,
```

Comment:
Why do we need to add a `db` argument to a method of `TypeInferenceBuilder`? It should already be available as `self.db()`